python/mpt/src/mpt/ethereum_tries.py

```diff
@@ -370,6 +370,23 @@ def from_data(data: Dict[str, Any]) -> State:
                 # If the account is not present in the preState data, we want it explicitly set to NONE instead of being a non-existent entry.
                 # This is very important for the args_gen purpose, as we need all touched accounts to be present in the initial state dict.
                 pre_state._main_trie._data[address] = None
+
+                # If this account has storage that will be accessed or initialized, we need to add it to the storage tries.
+                # Find the access list entry for this address if it exists
+                access_entry = next(
+                    (
+                        entry
+                        for entry in data["extra"]["accessList"]
+                        if entry["address"] == address_hex
+                    ),
+                    None,
+                )
+                if access_entry:
+                    for storage_key in access_entry["storageKeys"]:
+                        storage_key_bytes = Bytes32.fromhex(storage_key[2:])
+                        pre_state._storage_tries[address]._data[
+                            storage_key_bytes
+                        ] = None
```

Comment:
hmm ok so this is the case where an account is null before, null after, but we sload something?

Reply:
no it's the case where the account is null before, and we sload before we sstore something